locales/en/app.ftl

```diff
@@ -464,7 +464,7 @@ frequently-asked-questions=Frequently Asked Questions
 
 about-firefox-monitor = About {-product-name}
 mozilla-dot-org = Mozilla.org
-terms-and-privacy = Terms & Privacy
+terms-and-privacy-v2 = Terms & Privacy
```

Comment:
Interesting... What's the difference between this and the previous string, or, why can't we use the same string in both places?

```sh
$ cat ./locales/en/app.ftl | grep "terms-and-privacy"

terms-and-privacy = Terms and Privacy
terms-and-privacy-v2 = Terms & Privacy
```

Reply:
The ampersand in `terms-and-privacy-v2` is the only difference. After V2 is released, `terms-and-privacy` (no ampersand) will be retired. I think it makes more sense to just use the `terms-and-privacy` string we already have, but I need to double-check that content/ux does not have any strong feelings about this. Or ,@flodolo can we just update the existing `terms-and-privacy` string to use the ampersand for english only?

Reply:
> Or ,@flodolo can we just update the existing `terms-and-privacy` string to use the ampersand for english only?

Yes, that would be absolutely OK. In the end, it's just a styling thing for en-US

Reply:
Side note: you can move the existing string here, if it helps.